deploy/scripts/src/ipv-core/test.ts

```diff
@@ -227,220 +301,203 @@ const stubCreds = {
+        res.submitForm({
+          params: { redirects: 0 }
+        }),
+      { isStatusCode302 }))
+      res = group(groups[24].split('::')[1], () => timeRequest(() => // 02_KBVStub
+        http.get(res.headers.Location),
+      { isStatusCode200, ...pageContentCheck('Knowledge Based Verification (Stub)') }))
+    }, {})
   })
 
   sleepBetween(0.5, 1)
 
-  group('B01_Passport_10_KBVDataContinue POST', () => {
-    res = timeRequest(() => res.submitForm({
-      fields: {
-        jsonPayload: kbvPayloadP,
-        verificationScore: '2'
-      },
-      params: {
-        redirects: 0,
-        tags: { name: 'B01_Passport_10_KBVDataContinue_01_KBVStub' }
-      }
-    }),
-    { isStatusCode302 })
-    res = timeRequest(() => {
-      const response = http.get(res.headers.Location, { tags: { name: 'B01_Passport_10_KBVDataContinue_02_CoreCall' } })
-      // if (env.staticResources) getStaticResources(response)
-      return response
-    },
-    { isStatusCode200, ...pageContentCheck('You’ve successfully proved your identity') })
+  group(groups[25], () => { // B01_Passport_10_KBVDataContinue
+    timeRequest(() => {
+      res = group(groups[26].split('::')[1], () => timeRequest(() => // 01_KBVStub
+        res.submitForm({
+          fields: {
+            jsonPayload: kbvPayloadP,
+            verificationScore: '2'
+          },
+          params: { redirects: 0 }
+        }),
+      { isStatusCode302 }))
+      res = group(groups[27].split('::')[1], () => timeRequest(() => // 02_CoreCall
+        http.get(res.headers.Location),
+      { isStatusCode200, ...pageContentCheck('You’ve successfully proved your identity') }))
+    }, {})
   })
 
   sleepBetween(0.5, 1)
 
-  group('B01_Passport_11_ContinuePYISuccessPage POST', () => {
-    res = timeRequest(() => res.submitForm({
-      params: {
-        redirects: 0,
-        tags: { name: 'B01_Passport_11_ContPYISuccessPage_01_CoreCall' }
-      }
-    }),
-    { isStatusCode302 })
-    res = timeRequest(() => http.get(res.headers.Location, {
-      headers: { Authorization: `Basic ${encodedCredentials}` },
-      tags: { name: 'B01_Passport_11_ContPYISuccessPage_02_OrchStub' }
-    }),
-    { isStatusCode200, ...pageContentCheck('User information') })
-    iterationsCompleted.add(1)
+  group(groups[28], () => { // B01_Passport_11_ContinuePYISuccessPage
+    timeRequest(() => {
+      res = group(groups[29].split('::')[1], () => timeRequest(() => // 01_CoreCall
+        res.submitForm({
+          params: { redirects: 0 }
+        }),
+      { isStatusCode302 }))
+      res = group(groups[30].split('::')[1], () => timeRequest(() => // 02_OrchStub
+        http.get(res.headers.Location, {
+          headers: { Authorization: `Basic ${encodedCredentials}` }
+        }),
+      { isStatusCode200, ...pageContentCheck('User information') }))
+    }, {})
   })
+  iterationsCompleted.add(1)
+  console.log(userId + ',' + testEmail)
```

Comment:
Why has this debug statement been added?

Reply:
This was added to validate the ID reuse script and has been removed now